index.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
         "node": true
     },
     "parserOptions": {
-        "ecmaVersion": 2018
+        "ecmaVersion": 2020
```

Comment:
Not sure about that - we still run on Nodejs 10. Does that support ecma from 2020? What ES version is that?

Reply:
It supports node 10.8 and higher (our dependencies also switched to use that).

Reply:
(It's not the ecma from this year)

Reply:
In that case ok.